src/main/webapp/navbar.html

```diff
@@ -0,0 +1,30 @@
+<html>
+  <head>
+    <script src="https://unpkg.com/vue"></script>
+    <script src="https://unpkg.com/http-vue-loader"></script>
+  </head>
+  <body>
+    <div id="my-app">
+      <navigation :user="user"></navigation>
+    </div>
+    <script>
+      new Vue({
+        el: '#my-app',
+        components: {
+          'navigation': httpVueLoader('components/navbar.vue')
+        },
+        props: {
+          user: {
+            type: Object,
+            default: function() {
+              return {
+                name: 'Bob the Builder',
+                points: 597
+              }
+            }
+          }
+        }
```

Comment:
I've never seen props being set for the Vue instance itself before, since there's usually no parent that passes down any information. I might see this be useful if we start using vue's navigation implementation, though. If this prop is only being made for its default value, however, I recommend simply making the object a part of data.
```suggestion
        data: {
          user: {
            name: 'Bob the Builder',
            points: 597
          }
        }
```

Reply:
+1

Reply:
I see, I think I misunderstood how to use props. I'll change it to data instead.